src/test/java/core/basesyntax/service/operations/BalanceHandlerTest.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax.service.operations;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.Before;
+import org.junit.Test;
+
+public class BalanceHandlerTest {
+
+    @Before
+    public void before() {
+        Storage.getFruits().clear();
+    }
+
+    @Test
+    public void handle_correctNumberOfFruits_ok() {
+        FruitDao fruitDao = new FruitDaoImpl();
```

Comment:
Make tested dao class-level field - you use it in all methods.

**Same for other test classes.**